### ISSUE TITLE
Run error

### ISSUE THREAD

**Issue by xiaoyi-st** [thstkdgus35/EDSR-PyTorch#284]
Hello, use your code to debug the DIK2 data and found that the data cannot be loaded. Pytorch has also been upgraded to 1.1.0, python==3.6. There is no problem with the operating environment, but why the following error keeps appearing and the data cannot be loaded? The error is as follows:
**ModuleNotFoundError: No module named 'data.div2k_train_hr'**
Urgently need your help

**Comment by scutlrr** [comment]
the 'data_train' in option.py should be 'DIV2k'.

**Comment by xiaoyi-st** [comment]
> option.py中的“ data_train”应为“ DIV2K”。
I tried it, but I still get an error. I can’t find the relevant module, but my running environment is exactly the same as the project. my e-mail:xting120@qq.com   thank u

**Comment by xiaoyi-st** [comment]
> DIV2K

But immediately this error occurred ValueError: num_samples should be a positive integer value, but got num_samples=0

**Comment by scutlrr** [comment]
It may be attributed to the incorrect setting of 'data_dir'(in option.py). 
Assuming your dataset root is /home/USER_NAME/dataset/DIV2K， make sure data_dir=/home/USER_NAME/dataset
In addition, the directory structure of DIV2K is:
  DIV2K_train_HR
  DIV2K_train_LR/X4

**Comment by xiaoyi-st** [comment]
How can I train my own data set?




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "lrr"<notifications@github.com&gt;; 
发送时间: 2020年11月5日(星期四) 下午3:43
收件人: "thstkdgus35/EDSR-PyTorch"<EDSR-PyTorch@noreply.github.com&gt;; 
抄送: "R"<1120922880@qq.com&gt;; "Author"<author@noreply.github.com&gt;; 
主题: Re: [thstkdgus35/EDSR-PyTorch] Run error (#284)





 
It may be attributed to the incorrect setting of 'data_dir'(in option.py).
 Assuming your dataset root is /home/USER_NAME/dataset/DIV2K， make sure data_dir=/home/USER_NAME/dataset
 In addition, the directory structure of DIV2K is:
 DIV2K_train_HR
 DIV2K_train_LR/X4

**Comment by xiaoyi-st** [comment]
> 如何训练自己的数据集？
> […](#)
> ------------------＆nbsp;原始邮件＆nbsp; ------------------发件人：“ lrr” < notifications@github.com> ;; 发送时间：2020年11月5日（星期四）下午3:43收件人：“ thstkdgus35 / EDSR-PyTorch” <EDSR-PyTorch@noreply.github.com> ;; 抄送：“ R” <1120922880@qq.com> ;; “作者” <author@noreply.github.com> ;; 主题：回复：[thstkdgus35 / EDSR-PyTorch]运行错误（[＃284](https://github.com/thstkdgus35/EDSR-PyTorch/issues/284)）可能归因于'data_dir'（在option.py中）的错误设置。假设您的数据集根为/ home / USER_NAME / dataset / DIV2K，请确保data_dir = / home / USER_NAME / dataset此外，DIV2K的目录结构为：DIV2K_train_HR DIV2K_train_LR / X4 —之所以收到此消息是因为您编写了线程。直接回复此电子邮件，在GitHub上查看或取消订阅。
How can I train my own data set?

**Comment by scutlrr** [comment]
A simple method is that you can also create a directory named DIV2K, and then put your HR pictures into the DIV2K_train_HR directory, and put LR into the DIV2K_train_LR/X4 directory. Otherwise, you need to modify some parameters in div2k.

**Comment by xiaoyi-st** [comment]
> 一种简单的方法是，您还可以创建一个名为DIV2K的目录，然后将您的HR图片放入DIV2K_train_HR目录，然后将LR放入DIV2K_train_LR / X4目录。否则，您需要在div2k中修改一些参数。

I cannot set the DIV2K_train_LR / X4 directory, the'/' does not conform to the folder naming rules

**Comment by scutlrr** [comment]
This is a secondary catalog, which means that the DIV2K_train_LR catalog is created first, and then the X4 catalog is created.

**Comment by xiaoyi-st** [comment]
> 这是辅助目录，这意味着首先创建DIV2K_train_LR目录，然后再创建X4目录。

yes ,I did it, but it still appears: ModuleNotFoundError: No module named'data.divk2'. I use DIV2K data

**Comment by wusuhuang** [comment]
> 
DIV2K_train_LR/X4   change as   DIV2K_train_LR_bicubic/X4
